pkg_security.Rmd

```diff
@@ -44,7 +44,7 @@ Now, you might need to share secrets with CI services such as Travis or GitHub A
 
 * You could store API keys as environment variables / secrets, referring to the docs of the CI service.
 
-* For bigger secrets like tokens, refer [to the googlesheets guidance](https://cran.r-project.org/web/packages/googlesheets/vignettes/managing-auth-tokens.html#the-making-of-googlesheets.tar.gz-for-cran) that involves in particular the encryption of tokens for sharing them with Travis. Document the steps you made in a Markdown file so you, or say a new maintainer, can remember how to do that next time.
+* For bigger secrets like tokens, refer [to the gargle guidance](gargle.r-lib.org/articles/articles/managing-tokens-securely.html) that involves in particular the encryption of tokens for sharing them with Travis. Document the steps you made in [CONTRIBUTING.md](#friendlyfiles) so you, or say a new maintainer, can remember how to do that next time.
```

Comment:
Very minor point, but what does "bigger secrets" mean / are tokens really "bigger secrets" than API keys?
More generally, is the approach in the gargle vignette only applicable to tokens or also more generally to all types of secrets?

Reply:
very good point, I'll rephrase!

Reply:
thank you!